bee-common/bee-packable/src/error.rs

```diff
@@ -45,6 +45,46 @@ impl<U> UnpackError<Infallible, U> {
     }
 }
 
+/// Error type raised when `Packable::pack` fails.
+#[derive(Debug)]
+pub enum PackError<T, P> {
```

Comment:
Please don't hate me but can we put the type and impls of Pack before those of Unpack ?

Reply:
#ocd

Reply:
sure! makes total sense